plugins/xcmetrics/api-report.md

```diff
@@ -0,0 +1,23 @@
+## API Report File for "@backstage/plugin-xcmetrics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// @public (undocumented)
+export const XcmetricsPage: () => JSX.Element;
+
+// @public (undocumented)
+export const xcmetricsPlugin: BackstagePlugin<
```

Comment:
Nitpicking on the casing of xcmetrics in these two 😄 

I like `XcmetricsPage` and `XcmetricsPlugin`.

Reply:
I think however you'd get complaints from eslint everywhere as it's not being used as a constructor.

Reply:
And the other is a React Component so it /does/ make sense.

Reply:
I was not really sure about these. I looked at some other plugins (e.g. todo and techdocs) and saw that they use the same casing as I did. My thinking is that `xcmetricsPlugin` is a normal variable (camel case) while `XcmetricsPage` is a component (pascal case). What do you think about that reasoning?